r-package/grf/R/plot.R

```diff
@@ -80,10 +80,15 @@ export_graphviz <- function(tree) {
 }
 
 #' Plot a GRF tree object.
+#' 
 #' @param x The tree to plot
 #' @param ... Additional arguments (currently ignored).
 #'
 #' @method plot grf_tree
+#' @examples 
```

Comment:
Great, but see here how to use the examples section: https://github.com/grf-labs/grf/blob/master/r-package/grf/R/causal_forest.R#L79

i.e. please add a \dontrun and a full example that can be copy pasted and run